docs/writing-stories/loaders.md

```diff
@@ -22,6 +22,7 @@ Loaders are helpful when you need to load story data externally (e.g., from a re
     'vue/loader-story.3.js.mdx',
     'vue/loader-story.ts-3.ts.mdx',
     'angular/loader-story.ts.mdx',
+    'web-components/loader-story.ts.mdx',
```

Comment:
@kylegach, sorry for missing this earlier and correct me if I'm wrong, but you're referencing the `web-components/loader-story.ts.mdx`, but the file you've added is `web-components/loader-story.js.mdx`. Shouldn't it be mentioned as such and probably we'll need to add the TS variant as well? Don't you think?

Reply:
Good catch! 🙏